modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -11,6 +11,7 @@
     $mirror_name = 'mirrors.ocf.berkeley.edu',
     $mirror_path = "/opt/mirrors/ftp/${title}",
     $project_path = "/opt/mirrors/project/${title}",
+    $use_systemd = false,
```

Comment:
I'd change the default here to be `true` since you're changing it elsewhere to be passed in as such. If we have problems, then this'll provide an easy way to change back, but having it set to a default of true seems like it would make more sense for any future additions.